library/src/webapp/skin/neo-ox/portal.css

```diff
@@ -1448,7 +1448,9 @@ ul#otherSiteList li a span, ul.otherSitesCategorList li a span, ul.otherQuickLin
     background-position: -162px -252px;
 }
 .icon-sakai-sitebrowser {
-    background-position: -468px -522px;
+    background-position: center;
+    background-repeat: initial;
+    background-image: url(/library/image/silk/page_white_find.png) !important
```

Comment:
Could we move this out of the silk icon into the skin folder as well, as it shouldn't have ever been in there in the first place. Also I don't think you need the `!important`